src/pdn/src/grid.cpp

```diff
@@ -1260,6 +1263,14 @@ ShapeTreeMap InstanceGrid::getInstancePins(odb::dbInst* inst)
     if (net != nullptr) {
       for (auto* mpin : iterm->getMTerm()->getMPins()) {
         for (auto* box : mpin->getGeometry()) {
+          if (box->getTechLayer() == NULL) {
```

Comment:
How is this possible? I would think there is a bug somewhere else that allows this macro from being read in without the proper layer information. I don't mind the solution here, but I think @maliberty should take a look at ODB to see if it should have errored out or provided a warning there.

Reply:
A warning or error closer to the root cause would certainly help.

Reply:
I tried it and it my flow crashes a bit further down the road. Again with a layer being null, is my guess...

```
[INFO GPL-0075] Routability numCall: 1 inflationIterCnt: 1 bloatIterCnt: 0
Signal 11 received
Stack trace:
 0# 0x0000000000CA7D07 in /OpenROAD-flow-scripts/tools/install/OpenROAD/bin/openroad
 1# 0x00007FDFFD838400 in /lib64/libc.so.6
 2# odb::dbTechLayer::getType() in /OpenROAD-flow-scripts/tools/install/OpenROAD/bin/openroad
 3# grt::GlobalRouter::findInstancesObstructions(odb::Rect&, std::vector<int, std::allocator<int> > const&) in /OpenROAD-flow-scripts/tools/install/OpenROAD/bin/openroad
 4# grt::GlobalRouter::computeObstructionsAdjustments() in /OpenROAD-flow-scripts/tools/install/OpenROAD/bin/openroad
 5# grt::GlobalRouter::applyAdjustments(int, int) in /OpenROAD-flow-scripts/tools/install/OpenROAD/bin/openroad
 6# grt::GlobalRouter::initFastRoute(int, int) in /OpenROAD-flow-scripts/tools/install/OpenROAD/bin/openroad
```